records_mover/records/schema/field/pandas.py

```diff
@@ -57,7 +57,23 @@ def refine_field_from_series(field: 'RecordsSchemaField',
         field_type = field.python_type_to_field_type(unique_python_type)
         if field_type is not None:
             if RecordsSchemaField.is_more_specific_type(field_type, field.field_type):
-                field.field_type = field_type
+                if field.constraints is None:
+                    constraints = None
+                else:
+                    constraints = field.constraints.downcast(field_type)
+                field = RecordsSchemaField(name=field.name,
+                                           field_type=field_type,
+                                           constraints=constraints,
+                                           # TODO: Can I get this
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker